### PR TITLE
Update 2022-11-substarte-newsletter-anagolay

### PR DESCRIPTION
**Pull request by Eltairovy**
Suggestion for community highlights from Anagolay team

### CODE REVIEW

File: draft/2022-11-substrate-newsletter.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ $title
 ## TL;DR and important announcements
 
 ## 🔦 Community highlights
+
+
+
+[Anagolay team](https://anagolay.network/) has built [web3-api-auth-token library](https://github.com/kelp-hq/oss/tree/22f85a75887ddcf65fe411e008f9bc7ba2d3203f/tools/web3-api-auth-token). Web3 API Auth Token (WAAT) defines a compact self-contained and tamper-proof way for securely transmitting information between the parties as a base64Url encoded JSON object. Compared to standard JWT, WAAT is replacing a centralized server with a decentralized and self-governed component that is always Strategy specific, which is the base for truly decentralized tokens.
+
+WAATs can come in handy for:
```
Comment (sacha-l) [inline draft/2022-11-substrate-newsletter.md:12]:
```suggestion
WAAT can be used for for authorization and informational exchange, learn more [here](https://github.com/kelp-hq/oss/tree/22f85a75887ddcf65fe411e008f9bc7ba2d3203f/tools/web3-api-auth-token#why-we-should-use-waat).
```

File: draft/2022-11-substrate-newsletter.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ $title
...
+- Authorization: DApps can load the users' account from the wallet browser extension, use the keys to create the signature and then make the request to the API allowing users to access resources, services and routes that are permitted with that token.
+- Information Exchange: Web3 API Auth Tokens are an excellent way to transmit information where the integrity needs to be checked and validated. Because WAATs can be signed, any party receiving them can verify the integrity and authenticity. [Learn more about WAAT here](https://github.com/kelp-hq/oss/tree/22f85a75887ddcf65fe411e008f9bc7ba2d3203f/tools/web3-api-auth-token#why-we-should-use-waat).
+
```
Comment (sacha-l) [inline draft/2022-11-substrate-newsletter.md:15]:
```suggestion
```

**Review by sacha-l — APPROVED** [review]
Made this a little more concise / shorter.

**Comment by sacha-l** [comment]
Thanks for submitting this @Eltairovy :)
